### DISCUSSION TITLE
Multi repo milestones

### DISCUSSION THREAD

**Discussion by Salles-FA** [community/community#2945] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello gr8 people doing gr8 features for GitHub.

I'm wondering when we'll have the multi repo milestone functionality. When I googled it I found this https://github.com/dear-github/dear-github/issues/93... is it something on your roadmap? Could you add pls?? 

Thx! Cheers!!

**Comment by becca** [comment, marked as answer]
Right now this repository is exclusive to Discussions and GitHub for mobile feedback.

Did you envision this request playing into the Discussions feature in a particular way or find something specifically painful right now due to the lack of this feature as you start to use Discussions?

**Reply by Salles-FA** [reply]
Hi! 

I thought this repository "feedback" was open for discussions regarding all features on GitHub. I'm dealing with a multi repo project and we are struggling to deal with correlated issues within them. 
- https://github.com/tranquilitybase-io

Currently we are addressing Epics as issues in a "main" repo's milestone (e.g. https://github.com/tranquilitybase-io/TranquilityBase/milestone/1) and trying to link all issues from other repos to them... Also we're trying to put all related issues in a Project board at the organization level...

My current pain is that I'd like to have some kind of synchronization of the Milestone among all repos... or I'd like to have a Milestone at the organization level... 

Do you have any recommendation?

**Reply by Salles-FA** [reply]
https://github.com/githubevents/universe2020/discussions/604#discussioncomment-189056

**Comment by ghost** [comment]
Duplicate of https://github.com/github-community/community/discussions/6296 (as this has more up-votes).
